core/src/main/java/org/apache/accumulo/core/security/Authorizations.java

```diff
@@ -385,4 +385,21 @@ public String serialize() {
 
     return sb.toString();
   }
+
+  /**
+   * Converts to an Accumulo Access Authorizations object.
+   *
+   * @since 3.1.0
+   */
+  public org.apache.accumulo.access.Authorizations toAccessAuthorizations() {
+    if (auths.isEmpty()) {
+      return org.apache.accumulo.access.Authorizations.of(Set.of());
```

Comment:
Looking at the impl of `Set.of()` in my ide it does not allocate any objects, it always returns a statically initialized immutable empty set. Then with the eventual changes in accumulo-access this line will do no obj allocations, however it does that in a very round about way.   

Should probably add a `Authorizations.of()` method to follow javas conventions.  Also would make it consistent w/ `AccessExpression.of()`.  If `Authorizations.of()` existed then this could be more straightforward.

Reply:
Added at constant iin 5b825ab, seemed the best options w/ the current constraints of the beta API.  Can easily change it later.